HW03/Dmitry_Susha/dmitry_susha_hw_03_t_04.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task4(string, result = {})
```

Comment:
```suggestion
  def task4(string)
```

try not to change it